### PR TITLE
Setup CSS bundling, add bootstrap/normalize stylesheets

### PR DESCRIPTION
**Pull request by Arthelon**
This PR addresses #1 

- Sets up CSS bundling using Svelte preprocessor hooks
- Adds stylesheet for bootstrap-grid and normalize.css

### CODE REVIEW

File: src/template.html
Comment (kiootic) [inline src/template.html]: I'm not familiar with Svelte/Sapper. Do you know whether it supports bundling CSS stylesheets? We'd like to bundle these stylesheets (as well as our custom styles) directly as one CSS file, so that these external dependency is not needed.

**Review by kiootic — COMMENTED** [review]
Let's hold this PR after #5 is merged, so that we can test the full CI flow.

File: src/template.html
Comment (Arthelon) [reply src/template.html:10]: Yes it does support bundling CSS files. I will make this change immediately.

Comment (louischan-oursky) [reply src/template.html]: It is OK to reference external third CSS files. I am not sure how are we going to use the grid CSS framework. If the grid CSS framework is written in SCSS mixin, then we are going to bundle our CSS. If the grid CSS framework is just a CSS stylesheet, it is fine to reference it with `<link>`. Anyway, we need to bundle SCSS/CSS.

Comment (Arthelon) [reply src/template.html]: Turns out the styles for bootstrap grid system has its own CSS file. I've bundled it with normalize.css in my latest commit.

**Comment by kiootic** [comment]
@Arthelon Please rebase on master branch.

File: src/routes/_layout.svelte
Comment (kiootic) [inline src/routes/_layout.svelte]:
```suggestion
  import "../components/GlobalCSS.svelte";
```

By the way, if we want to disable some lint rule, we should specify which specific lint rule we are disabling like ` // eslint-disable-line no-use-before-define` and document the reason so that maintainer can know why it is disabled.

**Review by kiootic — COMMENTED** [review]
I'm getting this error when running `npm run dev`:
```
✗ client
'import' and 'export' may only appear at the top level
11: }
12: 
13: export function connect(port) {
    ^
14:   if (source || !window.EventSource) return;
```

Do you know why/how to fix it?

File: rollup.config.js
Comment (louischan-oursky) [inline rollup.config.js]: Since we now have browserlist, [let's not set `targets`](https://babeljs.io/docs/en/babel-preset-env#browserslist-integration)

**Comment by Arthelon** [comment]
Seems to be an unresolved issue with rollup/sapper: https://github.com/sveltejs/sapper/issues/1151 . I will try to find a fix.

File: src/routes/_layout.svelte
Comment (kiootic) [inline src/routes/_layout.svelte]: I suppose we can just do this?
`import "../components/GlobalCSS.svelte";`

**Review by kiootic — APPROVED** [review]

**Pull request merged**
